Resources/Prototypes/Entities/Markers/Spawners/Random/crystal.yml

```diff
@@ -0,0 +1,19 @@
+- type: entity
+  name: Crystal Spawner
+  id: CrystalSpawner
+  parent: MarkerBase
+  components:
```

Comment:
put a suffix `[70]` to indicate that this has a random chance of 70.

Reply:
suffix added.